src/launcher/main.cpp

```diff
@@ -453,15 +485,25 @@ int main(int argc, const char** argv) {
             output = "jfr";
 
         } else if (arg == "--timeout" || arg == "--loop") {
-            params << "," << (arg.str() + 2) << "=" << args.next();
+            const char* value = args.next();
+            params << "," << (arg.str() + 2) << "=" << value;
+            timeout = atoi(value);
```

Comment:
I see that timeout is initialized with `DEFAULT_FDTRANSFER_TIMEOUT`, but this prameter means the AP timeout, although we're not using it in this way.
I suggest you add a comment that this is meant for our usage of the `fdtransfer` action.